src/csharp/Grpc.IntegrationTesting/TestGrpc.cs

```diff
@@ -71,6 +71,13 @@ public static class TestService
         __Marshaller_SimpleRequest,
```

Comment:
I wasn't sure about that because they were tracked files. I can take them out of this PR if you think they don't need to be here. I figured maybe they were included so people trying to run the tests themselves didn't have to re generate the proto files